genestack_client/genestack_connection.py

```diff
@@ -32,6 +32,37 @@ def http_error_307(self, req, fp, code, msg, headers):
     http_error_301 = http_error_302 = http_error_303 = http_error_307
 
 
+class Response(object):
+    """Represents response from Genestack server."""
+
+    def __init__(self, data):
+        self._data = data
+
+    @property
```

Comment:
Maybe implement all these properties via `__getattr__()` instead of copypasting five times?